### DISCUSSION TITLE
Is the number of retries or timeout waiting for baseUrl configurable?

### DISCUSSION THREAD

**Discussion by tdreid** [cypress-io/cypress#14233] [CI setup]
I am using `cypress/included:6.1.0` in docker-compose to run end to end tests: 

When the test runner starts it will verify that a server can be reached at baseUrl. If not, it retries 3 times.

My services and web server need a little more time to start.

Can I increase the timeout and/or number of retries for this check.

Preferably, in my case I'd like a 'retry-until-success' policy, i.e. indefinite retries/wait.

I have checked the Timeouts section and the cypress.json documentation more generally. However none of those timeouts or retries seem to relate to this behavior.

Is there a setting for this?

---

Incidentally I asked a similar [question][1] on Stack Overflow, before finding this forum.

[1]:https://stackoverflow.com/questions/65358642/how-to-configure-cypress-to-wait-longer-or-indefinitely-for-baseurl

**Comment by jennifer-shehane** [comment]
No, this is not configurable. 

We recommend ensuring that your server is running before calling `cypress run`. Cypress's check on `baseUrl` is a final courtesy check just so you don't run through your whole test suite on a server that is not running.

For tips on ensuring your server is running before running Cypress read about booting your server in our docs: https://on.cypress.io/continuous-integration#Boot-your-server

**Reply by tdreid** [reply]
Thanks for confirming this @jennifer-shehane, as well for updating the SO. As a [feature request](https://github.com/cypress-io/cypress/issues/14262), I hope Cypress will consider (1) adding a flag to suppress this check and/or (2) allowing the number of retries to be configurable.
